index.bs

```diff
@@ -618,11 +618,11 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 				1. Let |r| be the result of running [=check clipboard read permission=] [=in parallel=]
```

Comment:
Should this still be in parallel? I guess not? Out of scope here anyway.

Reply:
agree... it's already "in parallel"..

```suggestion
				1. Let |r| be the result of running [=check clipboard read permission=]
```